.gitignore

```diff
@@ -118,3 +118,4 @@ env/
 
 ### Configuration files ###
 .env
+chromedriver.exe
```

Comment:
Nice spot on this -  unfortunately it won't override where a file has already been checked in, so chromedriver.exe is still committed to your repo (and this broke tests for me, because this particular version of chromedriver wouldn't play ball with my version of chrome)

Two options for fixing that:
* [See this SO answer](https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore) and use `git rm --cached`
* Remove this line from gitignore, and commit the removal of the chromedriver.exe binary, then add this line back in a subsequent commit

Reply:
Done